docs/40_smart-contract-guides/80_linking-custom-permission.md

```diff
@@ -2,72 +2,585 @@
+* Some knowledge of smart contracts, we recommend following the [hello world tutorial](../30_getting-started-guide/25_hello-world.md) tutorial
 
-[[info |Parent permission ]]
-| When you create a custom permission, the permission will always be created under a parent permission.
+## What is a Custom Permission
 
-If you have the authority of a parent permission which a custom permission was created under, you can always execute an action which requires that custom permission.
+When an account is created two permissions are created by default; `owner` and `active`. You can create a new permission, a custom permission, as a child permission of `owner`, `active` or another custom permission. Custom permissions require a public and private key pair. These custom permissions may be linked to an action of a smart contract and specify the permission required to execute that action. This permission system enables accounts to have flexible and highly granular control over accounts and smart contract actions.
 
-## Step 1. Create a Custom Permission
-Firstly, let's create a new permission level on the `alice` account:
+[[info]]
+| The permission `eosio.code` is a special permission which allows smart contracts to call inline actions. Smart contracts do not have access to account keys and the EOSIO blockchain performs an authorization check when an inline action is called. Adding `eosio.code` to an account permission gives explicit permission to execute code. For information on how to add the `eosio.code` see [cleos set account permission](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/set/set-account-permission.md)
+
+## Why use a Custom Permission
+
+Use custom permssions to improve the security on your account by setting a custom permission for calling specific transactions. This will limit the need to use the `owner` and `active` keys and limit the risk of these permissions being compromised.
+
+[[info]]
+| Custom permissions always have a parent permission. The parent permission has the authority to execute the same transactions as the custom permission, and to recover the custom permission keys if that custom permission is compromised.
+
+## Using Custom Permissions
+
+The following examples show you how to:
+
+* [Create a Custom Permission](#create-custom-permissions)
+* [Link a Custom Permission](#link-the-custom-permissions)
+* [Unlink a Custom Permission](#unlink-the-permission)
+* [Delete a Custom Permission](#delete-custom-permissions)
+
+In this example you will use:
+
+* An account called _bob_, and the keys to control this account stored in a local wallet.
+* An account called _scholder_, and the keys to control this account stored in a local wallet.
+* A smart contract called _hello_ which has been deployed to the _scholder_ account. This smart contract has three actions:
+	- `what(eosio::name user)`
+	- `why(eosio::name user)`
+	- `how(eosio::name user)`
```

Comment:
These three bullets will render on the same line. Use spaces instead of tabs, or just do as the other bulleted lists:
\* \`what(eosio::name user)\`
\* \`why(eosio::name user)\`
\* \`how(eosio::name user)\`

Reply:
updated to spaces